<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="vivek-PC" name="com.ThankYouProject.testCases.profile.TC_004_Profile" tests="2" failures="0" timestamp="15 Mar 2018 17:51:34 GMT" time="3.389" errors="1">
  <testcase name="userSaveProfile" time="0.003" classname="com.ThankYouProject.testCases.profile.TC_004_Profile">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ThankYouProject.utility.CommonFunctions.getLocator(CommonFunctions.java:66)
	at com.ThankYouProject.utility.CommonFunctions.getWebElement(CommonFunctions.java:58)
	at com.ThankYouProject.utility.CommonFunctions.click(CommonFunctions.java:36)
	at com.ThankYouProject.pages.Profile.userNotAbleToSubmitProfileBeforeSavingName(Profile.java:99)
	at com.ThankYouProject.testCases.profile.TC_004_Profile.userSaveProfile(TC_004_Profile.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- userSaveProfile -->
  <testcase name="userSaveProfile" time="3.386" classname="com.ThankYouProject.testCases.profile.TC_004_Profile">
    <skipped/>
  </testcase> <!-- userSaveProfile -->
</testsuite> <!-- com.ThankYouProject.testCases.profile.TC_004_Profile -->
